frontend/src/views/pages/Dashboard.vue

```diff
@@ -51,8 +51,8 @@
           </CCol>
         </CRow>
         <DashboardLineChart
-          v-if="selectedDealership"
-          :dealership="selectedDealership"
+          v-if="selectedDealership || $store.state.auth.role != 'Administration'"
+          :dealership="selectedDealership ? selectedDealership : $store.state.auth.dealership"
```

Comment:
This code change seems to be fixing the bug #102

Reply:
Can confirm the above.